AppZOO/AppZOO/IZooActions.cs

```diff
@@ -6,8 +6,24 @@ namespace AppZOO
 {
     interface IZooActions
     {
-        void BringToZoo(Zoo pSourceZoo, Animal pAnimal);
-        void TransferFromZoo(Zoo pDestinationZoo, Animal pAnimal);
+        /// <summary>
+        /// bring an animal from a source zoo to the current zoo
+        /// </summary>
+        /// <param name="pSourceZoo"></param>
+        /// <param name="pAnimal"></param>
+        void BringAnimal(Zoo pSourceZoo, Animal pAnimal);
+
+        /// <summary>
+        /// transfer an animal from the current zoo to a destination zoo
+        /// </summary>
+        /// <param name="pDestinationZoo"></param>
+        /// <param name="pAnimal"></param>
+        void TransferAnimal(Zoo pDestinationZoo, Animal pAnimal);
```

Comment:
foarte tare! 
Oare nu puteam gasi o metoda mai eleganta care sa ne ajute sa transferam. the la zoo curent, la alt zoo, un anume animal?
currentZoo.Transfer(destinationZoo, animal)
?

Reply:
iar in acest fel sacapm de BringAnimal, care face acelasi lucru, doar ca in sens invers?

Reply:
M-am orientat dupa cerintele din problema: "At a zoo they can **bring** new animals **and** **transfer** animals to new zoos". Cerinta mi-a dat de inteles ca ar trebui sa fie implementate ambele cerinte (bring si transfer) separat. Cel putin asa am inteles eu. Dar voi elimina BringAnimal in versiunea ulterioara a implementarii, asa cum e acum cerinta din PR.